app/forms/CASForm.scala

```diff
@@ -0,0 +1,15 @@
+package forms
+
+import model.CASLookup
+import play.api.data.Forms._
+import play.api.data._
+
+object CASForm {
+  def apply(): Form[CASLookup] = Form(
+    "cas" -> mapping(
+      "number" -> nonEmptyText,
+      "lastName" -> nonEmptyText,
```

Comment:
I know but really, I wanted to stick the the already existing CASService interface, which is a bit unfortunate:

https://github.com/guardian/membership-common/blob/6169b7460568104eb1940c63a67f0ea30cb56b61/src/main/scala/com/gu/subscriptions/CASApi.scala#L43

I could try to rewrite that, but is it worth it?